core/src/Slay/Prim.hs

```diff
@@ -68,12 +70,14 @@ deriving instance (Show (g (Maybe Natural)), Show (g (Color))) => Show (PrimText
 -- from -1 to 1
 newtype Curvature = Curvature Rational
 
+-- | curve direction
```

Comment:
This type can be used for other directions in the future (not only for curves), so I think this comment is redundant.